src/main/resources/environment.properties

```diff
@@ -2,4 +2,5 @@ browser=chrome
 hostname=127.0.0.1
 port=8080
 pageURL=/share/page/
-
+adminBrowsePanel=http://127.0.0.1:8080/share/page/console/admin-console/groups#state=panel%3Dsearch%26refresh%3Dfalse
```

Comment:
Do not repeat yourself. You already have constants with hostname, port and page url.

Reply:
Done